cmd/fission-bundle/main.go

```diff
@@ -64,7 +66,16 @@ func runRouter(ctx context.Context, clientGen crd.ClientGeneratorInterface, logg
 }
 
 func runExecutor(ctx context.Context, clientGen crd.ClientGeneratorInterface, logger *zap.Logger, mgr manager.Interface, port int) error {
```

Comment:
Can we create a based test? 

If we have `etcd` in envtest, we can create two instances of executor and see if one of them only runs.

```
k8s/1.23.5-darwin-arm64
etcd		kube-apiserver	kubectl
```